netci.groovy

```diff
@@ -5,6 +5,12 @@ import jobs.generation.*;
 def project = GithubProject
 def branch = GithubBranchName
 
+// 'isPullR' used as a parameter elsewhere in the file
+def isPullR = false
```

Comment:
Why define a new parameter rather than just passing the `isPR` parameter that already exists (and is set appropriately) on line 15?